package.json

```diff
@@ -4,11 +4,13 @@
   "description": "Arbitration Outsourcing for State Channels",
   "main": "index.js",
   "scripts": {
-    "test": "npx tsc && mocha build/__pisa_tests__ --exit",
+    "test": "npm run build && mocha $(find ./build/__pisa_tests__/*.js ! -path '*quickTest*') --exit",
+    "test-docker": "mocha ./build/__pisa_tests__/quickTest.js --exit",
     "build-kitsune": "cd statechannels && npm ci && npx truffle compile",
     "start-evm": "ganache-cli -d -l 15000000 --allowUnlimitedContractSize --defaultBalanceEther 1000000",
-    "start": "ts-node src/ts/service.ts",
-    "build": "npx tsc",
+    "start-dev": "ts-node src/startUp.ts",
+    "start": "node ./build/startUp.js",
+    "build": "mkdir -p ./build && cp ./config.json ./build/ && tsc",
```

Comment:
instead of copying config file, its better to move the config file out of the ts folder to the root folder and reference it with `../` then all references remain intact even after compilation

Reply:
agreed. I made another ticket about #50 , feel free to add any thoughts to that